src/components/Button/README.md

```diff
@@ -289,3 +289,120 @@ Use when a button has been pressed and the associated action is in progress.
+
+#### Navigation
+
+Merchants generally expect buttons to submit data or take action, and links to navigate. If navigation is required and the button component is passed a value in the `url` prop, the control will be outputted with an anchor styled as a button instead of a button in HTML to help convey this difference.
+
+For more information on making accessible links, see the [link component](/components/navigation/link).
+
+### Labeling
+
+The `accessibilityLabel` prop adds an `aria-label` attribute to the button, which can be accessed by assistive technologies like screen readers. Typically, this label text replaces the visible text on the button for merchants who use assistive technology.
+
+Use `accessibilityLabel` for a button if:
+
+- The button’s visible text doesn’t adequately convey the purpose of the button to non-visual merchants
+- The button has no text and relies on an icon alone to convey its purpose
+
+To help support merchants who use speech activation software as well as sighted screen reader users, make sure that the `aria-label` text includes any button text that’s visible. Mismatches between visible and programmatic labeling can cause confusion, and might prevent voice recognition commands from working.
+
+When possible, give the button visible text that clearly conveys its purpose without the use of `accessibilityLabel`. When no additional content is needed, duplicating the button text with `accessibilityLabel` isn’t necessary.
+
+<!-- usageblock -->
+
+#### Do
+
+```JSX
+<Button>Edit shipping address</Button>
+```
+
+```JSX
+<Heading>Shipping address</Heading>
+<Button accessibilityLabel="Edit shipping address">Edit</Button>
+```
+
+#### Don’t
+
+```JSX
+<Button accessibilityLabel="Change your shipping address">Edit</Button>
+```
+
+```JSX
+<Button accessibilityLabel="Edit">Edit</Button>
+```
+
+<!-- end -->
+
+#### External links
+
+If the button component is used to create a link to an external resource:
+
+- Use the `external` prop to make the link open in a new tab (or window, depending on the merchant’s browser settings).
+- Use the `icon` prop to add the `external` icon to the button.
+- Use the `accessibilityLabel` prop to include the warning about opening a new tab in the button text for non-visual screen reader users.
+
+For more information on making accessible links, see the [link component](/components/navigation/link).
+
+<!-- usageblock -->
+
+#### Do
+
+```JSX
+<Button accessibilityLabel="Terms and conditions (opens a new window)" icon="external" url="http://example.com" external>Terms and conditions</Button>
+```
+
+#### Don’t
+
+```JSX
+<Button url="http://example.com" external>Terms and conditions</Button>
+```
+
+<!-- end -->
+
+### Keyboard support
+
+Buttons use browser defaults for keyboard interactions.
+
+Give buttons keyboard focus with the `tab` key (or `shift` + `tab` when tabbing backwards)
```

Comment:
I realized we don't have a clean way to style the key info like the library does, since it's pulling from a `.md` file. Right now I've formatted this info to appear like other monospace text.

Reply:
We've been using the `<kbd>` tag, would this work?
```suggestion
Give buttons keyboard focus with the <kbd>tab</kbd> key (or <kbd>shift</kbd> + <kbd>tab</kbd> when tabbing backwards)
```

Reply:
Fixed with https://github.com/Shopify/polaris-styleguide/pull/2527!